File: Easy/Java/2351. First Letter to Appear Twice.java
class Solution {
    public char repeatedCharacter(String s) {
        Map<Character, Integer> count = new HashMap<>();
        char twice = 'a';

        for(int i = 0; i < s.length(); i++){
            if(!count.containsKey(s.charAt(i)))
                count.put(s.charAt(i), 1);
            else{
                twice = s.charAt(i);
                break;
            }
        }

        return twice;
    }
}

File: Easy/JavaScript/2351. First Letter to Appear Twice.js
var repeatedCharacter = function(s) {
    const count = new Map();
    let twice = 'a';

    for(let i = 0; i < s.length; i++){
        if(!count.has(s.charAt(i)))
            count.set(s.charAt(i), 1);
        else{
            twice = s.charAt(i);
            break;
        }
    }

    return twice;
};
